frontend/package.json

```diff
@@ -4,10 +4,10 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build:dev": "env-cmd -f .env.development next build && next export",
-    "build:stage": "env-cmd -f .env.staging next build && next export",
-    "build:prod": "env-cmd -f .env.production next build && next export",
-    "build:ui": "env-cmd -f .env.ui next build && next export",
+    "build:dev": "env-cmd -f .env.development next build && next start",
+    "build:stage": "env-cmd -f .env.staging next build && next start",
+    "build:prod": "env-cmd -f .env.production next build && next start",
```

Comment:
Please remove these `&& next start` and add a command `"start": "next start"`, or edit Dockerfile.cloud to add a ` &` to the end of the `npm run build:{...` command.

Reply:
Do we need to do both of these things?  I've adjusted the build commands to remove `next start` from them -- do I need to add `& npm run start` to the Dockerfile.cloud RUN command as well?

```json
"scripts": {
    "build:dev": "env-cmd -f .env.development next build",
    "build:stage": "env-cmd -f .env.staging next build",
    "build:prod": "env-cmd -f .env.production next build",
    "build:ui": "env-cmd -f .env.ui next build",
    "start": "next start",
  },
```